tests/cpp-tests/Classes/Sprite3DTest/Sprite3DTest.cpp

```diff
@@ -893,7 +893,7 @@ void AsyncLoadSprite3DTest::menuCallback_asyncLoadSprite(Ref* sender)
 
 void AsyncLoadSprite3DTest::asyncLoad_Callback(Sprite3D* sprite, void* param)
 {
-    auto index = (int32_t)param;
+    auto index = (long)param;
```

Comment:
`intptr_t` or `uintptr_t` is the best pointer conversion integer type

Reply:
Maybe `static_cast<int>((uintptr_t)param))` is better?

Reply:
Actually yes, that's a good point. I'll change it.